test/org/sagebionetworks/bridge/models/ForwardCursorPagedResourceListTest.java

```diff
@@ -29,15 +31,39 @@ public void canSerialize() throws Exception {
         accounts.add(new AccountSummary("firstName2", "lastName2", "email2@email.com", "id2", DateTime.now(),
                 AccountStatus.ENABLED, TestConstants.TEST_STUDY));
         
+        DateTime startTime = DateTime.parse("2016-02-03T10:10:10.000-08:00");
+        DateTime endTime = DateTime.parse("2016-02-23T14:14:14.000-08:00");
+        
         ForwardCursorPagedResourceList<AccountSummary> page = new ForwardCursorPagedResourceList<AccountSummary>(
-                accounts, "2", 100).withFilter("emailFilter", "filterString");
+                accounts, "nextOffsetKey")
+                .withRequestParam(ResourceList.OFFSET_KEY, "offsetKey")
+                .withRequestParam(ResourceList.START_TIME, startTime)
+                .withRequestParam(ResourceList.END_TIME, endTime)
+                .withRequestParam(ResourceList.SCHEDULED_ON_START, startTime)
+                .withRequestParam(ResourceList.SCHEDULED_ON_END, endTime)
+                .withRequestParam(ResourceList.PAGE_SIZE, 100)
+                .withRequestParam(ResourceList.EMAIL_FILTER, "filterString");
         
         JsonNode node = BridgeObjectMapper.get().valueToTree(page);
-        assertEquals(2, node.get("offsetKey").asInt());
+        
+        assertEquals("nextOffsetKey", node.get("offsetKey").asText());
+        assertEquals("nextOffsetKey", node.get("nextPageOffsetKey").asText());
+        assertEquals(startTime.toString(), node.get("startTime").asText());
+        assertEquals(endTime.toString(), node.get("endTime").asText());
+        assertEquals(startTime.toString(), node.get("scheduledOnStart").asText());
+        assertEquals(endTime.toString(), node.get("scheduledOnEnd").asText());
         assertEquals(100, node.get("pageSize").asInt());
-        assertEquals("filterString", node.get("emailFilter").asText());
```

Comment:
Added back. At some point in this process I realized I needed to keep testing deprecated properties, I missed this one.